sonic_platform_base/sonic_sfp/ext_media_handler_qsfp28.py

```diff
@@ -0,0 +1,162 @@
+########################################################################
+# DellEMC
+#
+# Module contains the ext media drivers for QSFP28 modules
+#
+########################################################################
+
+from ext_media_utils import *
+from ext_media_handler_base import media_static_info
+
+QSFP28_LENGTH_ADDR = media_eeprom_address(offset=146)
```

Comment:
why QSFP28 and QSFP+ need to be handled in different files? Shouldn't they all follow SFF8436/SFF8636? Btw, we already have an sff8436 parser, are all these new added here not available in the current SFF8436 parser? If yes could I suggest that to extend the current parser instead of adding a new separate parser?